specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -3957,6 +4165,110 @@
         }
       }
     },
+    "Addon": {
+      "description": "An addon resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of an addon resource",
+          "$ref": "#/definitions/AddonProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AddonUpdate": {
+      "description": "An update of an addon resource",
+      "properties": {
+        "properties": {
+          "description": "The properties of an addon resource that may be updated",
+          "$ref": "#/definitions/AddonUpdateProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AddonUpdateProperties": {
+      "description": "The properties of an addon that may be updated",
+      "properties": {
+        "properties": {
+          "description": "reference specific properties definition for addon type being used",
+          "$ref": "#/definitions/AddonProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AddonProperties": {
+      "description": "The properties of an addon that may be updated",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AddonSrmProperties"
+        }
+      ],
+      "properties": {
+        "systemData": {
+          "description": "required by CI process",
```

Comment:
Is this expected? "properties" included twice?

Reply:
This was already removed. `systemData` will be in the next API version, not this one. See commit 3c0ca92.